hphp/runtime/vm/jit/vasm-emit.cpp

```diff
@@ -134,7 +134,7 @@ void emitVunit(Vunit& vunit, const IRUnit& unit,
   CodeBlock cold;
   CodeBlock* frozen = &code.frozen();
 
-  auto const do_relocate = arch_any(Arch::X64, Arch::PPC64) &&
+  auto const do_relocate = arch_any(Arch::X64, Arch::PPC64, Arch::ARM) &&
```

Comment:
Just delete this first condition—now it's just checking that we're running on any architecture, which is trivially true...